docs/self-managed/installation-methods/helm/cloud-providers/amazon/amazon-eks/terraform-setup.md

```diff
@@ -718,6 +720,86 @@ Running these commands will clean up both the job and the secret, ensuring that
 </TabItem>
 </Tabs>
 
+### Accessing internal infrastructure
+
+:::warning Not recommended in production
+
+These approaches are intended for **development and troubleshooting purposes only**.
+For a production cluster, you should use a proper VPN or other secure access methods.
+
+:::
+
+Some infrastructure components (like OpenSearch dashboards or Aurora PostgreSQL databases) are only accessible from inside the EKS network.
+You can use a temporary pod as a **“jumphost”** to tunnel traffic to these components.
+
+---
+
+<details>
+<summary>Generic approach using a socat pod</summary>
+
+
+0. **Component connection details**
+
+   | Component            | Remote Host        | Remote Port    | Local Port     |
+   | -------------------- | ------------------ | -------------- | -------------- |
+   | OpenSearch dashboard | `$OPENSEARCH_HOST` | 443            | 9200           |
+   | Aurora PostgreSQL    | `$AURORA_ENDPOINT` | `$AURORA_PORT` | `$AURORA_PORT` |
+
+
+1. **Run a socat pod to create a TCP tunnel**
+
+   Replace `<LOCAL_PORT>`, `<REMOTE_HOST>`, and `<REMOTE_PORT>` with the component-specific values.
+
+   ```bash
+   kubectl --namespace $CAMUNDA_NAMESPACE run my-jump-pod -it \
+     --image=alpine/socat \
+     --tty --rm \
+     --expose=true --port=<REMOTE_PORT> \
+     tcp-listen:<REMOTE_PORT>,fork,reuseaddr \
+     tcp-connect:<REMOTE_HOST>:<REMOTE_PORT>
+   ```
+
+   :::tip How it works
+   [`socat`](http://www.dest-unreach.org/socat/) (*SOcket CAT*) is a command-line tool that relays data between two network endpoints.
```

Comment:
just a comment that the combo of `name` and [link](#) is not well visible.

<img width="170" height="83" alt="image" src="https://github.com/user-attachments/assets/42347a27-d526-413e-b88b-488e6b40b070" />

I think I'd have overlooked the link if I didn't see the markdown for reviewing.